core/rawdb/schema.go

```diff
@@ -92,6 +95,7 @@ var (
 	SnapshotAccountPrefix = []byte("a") // SnapshotAccountPrefix + account hash -> account trie value
 	SnapshotStoragePrefix = []byte("o") // SnapshotStoragePrefix + account hash + storage hash -> storage trie value
 	CodePrefix            = []byte("c") // CodePrefix + code hash -> account code
+	skeletonHeaderPrefix  = []byte("S") // skeletonHeaderPrefox + num (uint64 big endian) -> header
```

Comment:
Maybe use something that doesn't clash with the meta entries.

Reply:
Myeah, changing this would cause all merge testnet nodes to blow up since they'd need a database upgrade (i.e. resync). I don't really see it as valuable as how much a PITA it is to change.